setup.py

```diff
@@ -24,25 +24,24 @@
       install_requires=[
           "requests>=2.31.0,<3",
           "pyjwt>=2,<3",
-          "arrow>=0.15.4,<2",
-          "gemd>=1.15.0,<2",
-          "boto3>=1.17.93,<2",
-          "botocore>=1.20.95,<2",
+          "arrow>=1.3.0,<2",
+          "gemd>=1.16.7,<2",
```

Comment:
Is this just an instinctive set-to-latest, or is there a feature-based motivation for choosing this one?

Reply:
Just an instinctive bumping, since I feel like nudging users towards newer releases sooner rather than later is good practice. But please let me know if you'd rather I left them as is, since in these (and most) cases, it's just a minor bump, so should be fine left alone.